CONTRIBUTING.md

```diff
@@ -1,235 +1,198 @@
-How to Submit Patches for ovn-kubernetes
-========================================
+# Contributing Guide
 
-Create github pull requests for the repo.  More details are included below.
+* [New Contributor Guide](#contributing-guide)
+  * [Ways to Contribute](#ways-to-contribute)
+  * [Find an Issue](#find-an-issue)
+  * [Ask for Help](#ask-for-help)
+  * [Pull Request Lifecycle](#pull-request-lifecycle)
+  * [Development Environment Setup](#development-environment-setup)
+  * [Sign Your Commits](#sign-your-commits)
+  * [Pull Request Checklist](#pull-request-checklist)
 
-Before You Start
-----------------
+Welcome! We are glad that you want to contribute to our project! 💖
 
-Before you send patches at all, make sure that each patch makes sense.
-In particular:
+As you get started, you are in the best position to give us feedback on areas of
+our project that we need help with including:
 
-  - A given patch should not break anything, even if later
-    patches fix the problems that it causes.  The source tree
-    should still work after each patch is applied.  (This enables
-    `git bisect` to work best.)
+* Problems found during setting up a new developer environment
+* Gaps in our Quickstart Guide or documentation
+* Bugs in our automation scripts
 
-  - A patch should make one logical change.  Don't make
-    multiple, logically unconnected changes to disparate
-    subsystems in a single patch.
+If anything doesn't make sense, or doesn't work when you run it, please open a
+bug report and let us know!
 
-  - A patch that adds or removes user-visible features should
-    also update the appropriate user documentation or manpages.
+## Ways to Contribute
 
-Commit Summary
--------------
+We welcome many different types of contributions including:
 
-The summary line of your commit should be in the following format:
-`<area>: <summary>`
+* New features
+* Builds, CI/CD
+* Bug fixes
+* Documentation
+* Issue Triage
+* Answering questions on Slack/Mailing List
+* Web design
+* Communications / Social Media / Blog Posts
+* Release management
 
-  - `<area>:` indicates the area of the ovn-kubernetes to which the
-    change applies (often the name of a source file or a
-    directory).  You may omit it if the change crosses multiple
-    distinct pieces of code.
+Not everything happens through a GitHub pull request. Please come to our
+[meetings](./MEETINGS.md) or [contact us](https://ovn-org.slack.com/archives/C010SQ5FSNL) and let's discuss how we can work
```

Comment:
Link to meetings is broken.

Reply:
hmm I think its because the ordering of the commits are different...

Reply:
nope its because this renders this to "https://github.com/ovn-org/ovn-kubernetes/blob/project-governance/MEETINGS.md" but once we merge this PR it will work because it will render it to "https://github.com/ovn-org/ovn-kubernetes/blob/main/MEETINGS.md since currently that file doesn't really exist.